metrique/examples/timers.rs

```diff
@@ -49,7 +49,7 @@ type DefaultTimestampFormat = EpochMicros;
 
 // By implementing the `ValueFormatter` trait, you can also define your own entirely custom formats.
 
-#[metrics(prefix = "subevent_", subfield)]
```

Comment:
this has to be owned because `TimestampOnClose` mutates?

Reply:
we don't need to do it here, but we should add a UI test for `subfield_owned` when you need `subfield` (and ideally have a nice error)